### ISSUE TITLE
implement ape14 for wcs 

### ISSUE THREAD

**Issue by sosey** [spacetelescope/imexam#159]
To allow fits and asdf files to be displayed to ds9 and ginga using the appropriate wcs objects.

**Comment by pllim** [comment]
Ginga already has a PR for this over at ejeschke/ginga#719 . It basically works; I just don't have time right now to go back and wrap it up.

**Comment by sosey** [comment]
yes, but I need to implement it for DS9 and update my calls into ginga
